plugins/default/lib/disk_usage.py

```diff
@@ -53,6 +62,10 @@ def analize(pod, disks, warning_threshold, critical_threshold):
 
 
 def report(results):
+    # REVIEW: is nagios.UNKNOWN the best here?
```

Comment:
As we are no longer filtering out tmpfs mounts, if we aren't getting a result from df, something is seriously wrong.  The overall status reported to nagios is taking the highest numeric value, so just one missing df result will show as UNKNOWN.  I honestly can't forsee a situation where we would get one empty df result yet get all the rest just fine.